packages/ddl-shim/src/dermatology.ts

```diff
@@ -0,0 +1,229 @@
+
+    ddl2.dataviews.forEach(apply_to_dataview);
+
+    function apply_to_dataview(dv: any) {
+        const widgetId = dv.id;
+        const dermPanelObj: any = findKeyVal(dermObj, "__id", widgetId);
+        if (dermPanelObj) {
+            const dermPanelProps: any = dermPanelObj.__properties;
+            const dermWidgetObj: any = dermPanelObj.__properties.chart ? dermPanelObj.__properties.chart : dermPanelObj.__properties.widget;
+            const dermWidgetProps: any = dermWidgetObj.__properties;
+            apply_class_mapping(dermWidgetObj);
+            apply_panel_property_mapping(dermPanelProps, dermWidgetProps);
+            apply_widget_property_mapping(dermPanelProps, dermWidgetProps);
+            if (dv.visualization.properties.chartType) {
+                dv.visualization.properties.charttype = dv.visualization.properties.chartType;
+            }
+        } else {
+            console.warn(widgetId + " not found in dermObj");
+        }
+
+        function apply_class_mapping(dermWidgetObj: any) {
+            dv.visualization.__class = swap_with_supported_class(dermWidgetObj.__class);
+            dv.visualization.properties.__class = "marshaller_VizChartPanel";
+            if (!dv.visualization.properties.widget) dv.visualization.properties.widget = {};
+            dv.visualization.properties.widget.__class = dv.visualization.__class;
+
+            function swap_with_supported_class(_class: string): string {
+                return classMappings[_class] ? classMappings[_class] : _class;
+            }
+        }
+        function apply_panel_property_mapping(dermPanelProps: any, dermWidgetProps: any) {
+            dv.visualization.title = dermPanelProps.title || "";
+            dv.visualization.description = "";
+            dv.visualization.visibility = dv.visualization.visibility === "flyout" ? "flyout" : "normal";
+            dv.visualization.chartType = dv.visualization.__class.split("_")[1];
+            for (const propName in dermPanelProps) {
+                if (typeof propertyMappings[propName] !== "undefined") {
+                    const newPropName = propertyMappings[propName].name;
+
+                    if (typeof propertyMappings[propName].transform === "function") {
+                        dv.visualization.properties[newPropName] = propertyMappings[propName].transform(dermPanelProps[propName]);
+                    } else {
+                        dv.visualization.properties[newPropName] = dermPanelProps[propName];
+                    }
+
+                }
+            }
+            if (dermWidgetProps && dermWidgetProps.showLegend && dv.visualization.properties) {
+                dv.visualization.properties.legendVisible = true;
+            }
+        }
+        function apply_widget_property_mapping(dermPanelProps: any, dermWidgetProps: any) {
+            dv.visualization.title = dermWidgetProps.title ? dermWidgetProps.title : "";
```

Comment:
This line should be replaced with:
`if(dv.visualization.title === "")dv.visualization.title = dermWidgetProps.title || "";`